packages/webpack-config/web-default/index.html

```diff
@@ -40,7 +40,7 @@
         scroll-behavior: smooth;
         /* Prevent text size change on orientation change https://gist.github.com/tfausak/2222823#file-ios-8-web-app-html-L138 */
         -webkit-text-size-adjust: 100%;
-        height: 100%;
+        height: calc(100% + env(safe-area-inset-top));
```

Comment:
Hmm, interesting idea. Have you tested this behavior with react-native-safe-area-context (react-navigation), landscape mode on iOS (resizing header bar)? Could you please add a "test plan" to your PR description:

```
# Test Plan

...
```

Reply:
Yes it works! I added screenshots to the test plan showing the behavior with react-navigation bottom tabs and header, with SafeArea component and with landscape mode.